Makefile

```diff
@@ -30,8 +30,8 @@ validate:
 	$(DCRUN) cargo clippy --all-targets --all-features --no-deps -- -D warnings
 	$(DCRUN) cargo deny check
 
-## licenses:     Build license summary file
-licenses:
+## licenses-report: Build license summary file
+licenses-report:
```

Comment:
Make doesn't like it if the target has the same name as an existing file/directory. Easiest fix is to change the target name.